ops/ccs-ops-misc/synthetic-data/scripts/synthea-automation/generate-characteristics-file.py

```diff
@@ -29,18 +30,17 @@ def generate_characteristics_file(args):
     
     bene_id_start = args[0]
     bene_id_end = args[1]
-    db_data = args[2].split(',')
-    output_path = args[3]
-    env = args[4]
+    output_path = args[2]
+    env = args[3]
     
     db_string = ""
-    
+
     if "test" == env:
-        db_string = f"postgres://{db_data[0]}:{db_data[1]}@bfd-test-aurora-cluster.cluster-ro-clyryngdhnko.us-east-1.rds.amazonaws.com/fhirdb"
+        db_string = ssmutil.get_ssm_db_string("test")
     elif "prd-sbx" == env or "sbx" == env:
-        db_string = f"postgres://{db_data[0]}:{db_data[1]}@bfd-prod-sbx-aurora-cluster.cluster-ro-clyryngdhnko.us-east-1.rds.amazonaws.com/fhirdb"
-    elif "prd" == env:
-        db_string = f"postgres://{db_data[0]}:{db_data[1]}@bfd-prod-aurora-cluster.cluster-ro-clyryngdhnko.us-east-1.rds.amazonaws.com/fhirdb"
+        db_string = ssmutil.get_ssm_db_string("prod-sbx")
+    elif "prd" == env or "prod" == env:
```

Comment:
allowing 'prd' and 'sbx' variations doesn't seem necessary to me and could create confusion in other places. any compelling reason for it or can we standardize to one thing per?

Reply:
Noticed it's not consistent across BFD if prod is "prod" or "prd", and sbx was just to make it a bit shorter, but I can remove them if its confusing. Reminds me I should add a warning if an unknown value is put in, since it will just ignore them and if you put in something weird and not do what you probably wanted currently.

Reply:
Fixed to only accept the canonical env names, and added errors when a bad env is input for two scripts where it was missing.